samples/highcharts/demo/html-legend/demo.details

```diff
@@ -0,0 +1,10 @@
+---
+name: Line chart with HTML legend
+authors:
+  - Hubert Kozik
+js_wrap: b
+tags:
```

Comment:
`new_until: ` with some date could be added. The unwritten rule seems to be 3 months after the publish / release. With the vacation time at hand, maybe `2025-10-01`?